<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>250</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Gulim</family>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Ultimate Video Converter</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../Vid.ico</normaloff>../Vid.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">color:rgb(180, 180, 180);
alternate-background-color: rgb(0, 0, 0);
background-color: rgb(42, 42, 42);

selection-background-color: rgb(0, 0, 0);</string>
  </property>
  <property name="iconSize">
   <size>
    <width>32</width>
    <height>32</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="font">
    <font>
     <family>Gulim</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="1">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;Tahoma&quot;;
border: 0px solid;</string>
      </property>
      <property name="title">
       <string>Video Quality</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <widget class="QSpinBox" name="spinBox">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>40</width>
         <height>20</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;From 0 to 51. &lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;(0 - best quality, 51 - the worst quality)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;Tahoma&quot;;
background-color: rgb(255, 255, 255);
color:rgb(0, 0, 0);
</string>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::PlusMinus</enum>
       </property>
       <property name="suffix">
        <string/>
       </property>
       <property name="prefix">
        <string/>
       </property>
       <property name="maximum">
        <number>51</number>
       </property>
      </widget>
     </widget>
    </item>
    <item row="0" column="0" colspan="4">
     <widget class="QLabel" name="label">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Tahoma</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Ultimate Video Converter</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="1" colspan="2">
     <widget class="QPushButton" name="pushButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Tahoma</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Start Video Conversion&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	border-radius: 5px;
	border: 0.5px solid
}
QPushButton:hover {
    background-color: qlineargradient(spread:pad, x1:0.492, y1:1, x2:0.46995, y2:0, stop:0 rgba(42, 42, 42, 255), stop:1 rgba(72, 72, 72, 255));
}
QPushButton:pressed {
	
	background-color: rgb(0, 0, 0);
}
QToolTip {
    padding: 5px;
    border-radius: 3px;
}</string>
      </property>
      <property name="text">
       <string>Convert</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QGroupBox" name="groupBox_4">
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;Tahoma&quot;;
border: 0px solid;</string>
      </property>
      <property name="title">
       <string>Conversion Speed</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QComboBox" name="comboBox">
         <property name="font">
          <font>
           <family>Tahoma</family>
           <pointsize>12</pointsize>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;The slower the conversion &lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;the better the quality&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="styleSheet">
          <string notr="true">selection-color: rgb(255, 255, 255);
font: 12pt &quot;Tahoma&quot;;</string>
         </property>
         <property name="currentText">
          <string/>
         </property>
         <property name="insertPolicy">
          <enum>QComboBox::NoInsert</enum>
         </property>
         <property name="placeholderText">
          <string>None</string>
         </property>
         <item>
          <property name="text">
           <string>Ultra Fast</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Super Fast</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Very Fast</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Faster</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Fast</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Medium</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Slow</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Slower</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Very Slow</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QGroupBox" name="groupBox_3">
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;Tahoma&quot;;
border: 0px solid;</string>
      </property>
      <property name="title">
       <string>Video Tune</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QComboBox" name="comboBox_2">
         <property name="font">
          <font>
           <family>Tahoma</family>
           <pointsize>12</pointsize>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Choose Video Tune&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="styleSheet">
          <string notr="true">selection-color: rgb(255, 255, 255);
font: 12pt &quot;Tahoma&quot;;</string>
         </property>
         <property name="currentText">
          <string/>
         </property>
         <property name="insertPolicy">
          <enum>QComboBox::NoInsert</enum>
         </property>
         <property name="placeholderText">
          <string>None</string>
         </property>
         <item>
          <property name="text">
           <string>Film</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Animation</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Grain</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Still image</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Fast decode</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Zero Latency</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;Tahoma&quot; ;
border: 0px solid;</string>
      </property>
      <property name="title">
       <string>Encoding Codec</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QRadioButton" name="radioButton">
         <property name="font">
          <font>
           <family>Tahoma</family>
           <pointsize>12</pointsize>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;H264&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 12pt &quot;Tahoma&quot;;</string>
         </property>
         <property name="text">
          <string>Default</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_2">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;H265&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 12pt &quot;Tahoma&quot; ;</string>
         </property>
         <property name="text">
          <string>HEVC</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
   <zorder>label</zorder>
   <zorder>groupBox</zorder>
   <zorder>groupBox_2</zorder>
   <zorder>groupBox_4</zorder>
   <zorder>groupBox_3</zorder>
   <zorder>pushButton</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tahoma</family>
     <pointsize>10</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QMenuBar {
    background-color:qlineargradient(spread:pad, x1:0.492, y1:1, x2:0.46995, y2:0, stop:0 rgba(42, 42, 42, 255), stop:1 rgba(90, 90, 90, 255));
    spacing: 3px; /* spacing between menu bar items */
}

QMenuBar::item {
    padding: 1px 4px;
    background: transparent;
    border-radius: 4px;
}

QMenuBar::item:selected { /* when selected using mouse or keyboard */
    background: rgb(36, 36, 36);
}

QMenuBar::item:pressed {
    background: rgb(0, 0, 0);
}</string>
   </property>
   <widget class="QMenu" name="menuChoose_file">
    <property name="styleSheet">
     <string notr="true">QToolTip {
    padding: 5px;
    border-radius: 3px;
    opacity: 200;
}</string>
    </property>
    <property name="title">
     <string>File</string>
    </property>
    <property name="toolTipsVisible">
     <bool>true</bool>
    </property>
    <addaction name="actionOpen"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <property name="toolTipsVisible">
     <bool>true</bool>
    </property>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionSettings"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelp"/>
   </widget>
   <addaction name="menuChoose_file"/>
   <addaction name="menuSettings"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="sizeGripEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="toolTip">
    <string>Load new file </string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
